local_planner/src/nodes/planner_functions.cpp

```diff
@@ -24,6 +24,13 @@ void processPointcloud(
 
   float distance;
 
+  float fov_yaw_lower_boundary = fov.azimuth_deg - fov.h_fov_deg / 2.0f;
```

Comment:
maybe just to stay consistent, rename the variables here to `fov_azimuth_lower_boundary`, `fov_elevation_lower_boundary`, ...